### ISSUE TITLE
2.3 version update

### ISSUE THREAD

**Issue by a-am** [fruitstudios/LinkIt#25]
We added the new fruitlink plugin folder to the craft plugins folder.  The database shows the migration ran however the old version wasn't removed and the previous linkit inputs weren't updated to the new version.  Did we miss a step in the update process?

**Comment by a-am** [comment]
We are updating from version 0.9.1.  There are no errors in the logs.

**Comment by samhibberd** [comment]
Are you on slack? Is it possible to send over a db dump?

**Comment by samhibberd** [comment]
Just noticed an old 'linkit' folder snuck in, were you updating using that one? 2.3 is an entirely new plugin and needs to be installed from scratch. Just tested on your version locally and aok!

**Comment by samhibberd** [comment]
Hmm. And installed it?

**Comment by a-am** [comment]
Yes clicked install.

**Comment by a-am** [comment]
I tried it again and got this error. 
![screen shot 2016-03-10 at 4 59 18 pm](https://cloud.githubusercontent.com/assets/976370/13688808/78682040-e6e1-11e5-8308-bc5da62ada12.png)

**Comment by page-8** [comment]
Was just about to report this. Also got this error. Was able to fix it by updating the migration on line 60 to:

```
if (!empty($existingSettings['assetSources']))
                {
                    $existingSettings['assetSources'] = $existingSettings['assetSources'];
                }
```

**Comment by samhibberd** [comment]
Thanks @page-8, yes that would do it, although I am trying to work out in what situation `assetSources` would not be defined and whether i'm missing something important. 

Do you also have a copy of your pre Link It update database that I you would be able to share (over on Slack)?

**Comment by samhibberd** [comment]
Resolved in 2.3.2, thanks for your help chaps.
